pkg/controller/kedacontroller/transform/transform.go

```diff
@@ -205,6 +201,14 @@ func EnsureCertInjectionForDeployment(configMapName string, secretName string, s
 	}
 }
 
+func AddPathsToCerts(values []string, prefixes []Prefix, scheme *runtime.Scheme, logger logr.Logger) []mf.Transformer {
```

Comment:
I'd name this something like `EnsurePathsToCertsInDeployment` just because of consistency with other similar functions